components/contributors/InfoCard.tsx

```diff
@@ -24,31 +24,37 @@ export default function InfoCard({
       )}
       role="listitem"
     >
-      <div className="flex shrink-0 items-center space-x-4 md:space-y-6 xl:space-y-1 ">
-        <Link
-          href={isClickable ? `/contributors/${contributor.github}` : `#`}
-          className=""
-        >
-          <div
-            className={`z-10 flex h-28 w-28 shrink-0 items-center rounded-full md:h-32 md:w-32 md:p-1 ${
-              isClickable && `cursor-pointer`
-            }`}
+      <div className="flex shrink-0 items-center space-x-2 md:space-y-6 xl:space-y-1 ">
+        <div className="flex">
+          {!!rank && (
+            <div className="mr-5 flex h-20 items-center self-center text-5xl font-bold tracking-wider text-secondary-500 dark:text-secondary-400 lg:text-6xl">
+              #{rank}
+            </div>
+          )}
+          <Link
+            href={isClickable ? `/contributors/${contributor.github}` : `#`}
+            className=""
           >
-            <Image
-              className="rounded-full border-2 border-indigo-500"
-              src={`https://avatars.githubusercontent.com/${contributor.github}`}
-              alt={contributor.github}
-              height={112}
-              width={112}
-            />
-          </div>
-        </Link>
-        <div
-          className={`overflow-hidden ${
-            minimal ? "" : "flex flex-col items-center space-y-2"
-          }
-          `}
-        >
+            <div
+              className={`dark:border-1 z-10 mr-2 shrink-0 rounded-full border-2 border-current ${
+                rank != null
+                  ? ["text-yellow-600", "text-stone-600", "text-amber-700"][
+                      rank - 1
+                    ] ?? "text-purple-600"
+                  : "text-purple-600"
+              } rounded-full ${rank != null && rank > 0 && rank <= 3 && "animate-circular-shadow"}`}
```

Comment:
Create a reusable function that returns the desired classNames based on rank and re-use that for both in the leaderboard contributor image and here. This way it's guaranteed to always have same logic.

Reply:
![image](https://github.com/coronasafe/leaderboard/assets/99136186/58c5a47f-27e9-4b15-b37b-8ec5cce2afef)
Should I keep this function in the `InfoCard.jsx` or move it in `utils.js` ?

Reply:
utils